docs/install-helm.md

```diff
@@ -0,0 +1,37 @@
+# Install with Helm
+
+## Prerequisites
+
+* Kubernetes cluster with support for services of type `LoadBalancer`
+* [kubectl CLI](https://kubernetes.io/docs/reference/kubectl/)
+
+## Basic Installation
+
+The latest helm installation instructions for the kuadrant operator are maintained at [https://artifacthub.io/packages/helm/kuadrant/kuadrant-operator](https://artifacthub.io/packages/helm/kuadrant/kuadrant-operator).
+
+After installing the operator, you can create a Kuadrant resource to install the operand components.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1beta1
+kind: Kuadrant
+metadata:
+  name: kuadrant
+  namespace: kuadrant-system
+EOF
+```
+
+If everything went well, the status of the resource shoulde be `Ready`
+
+```bash
+kubectl get kuadrant kuadrant -n kuadrant-system -o=wide
+```
+
+```bash
+NAME       STATUS   AGE
+kuadrant   Ready    64s
+```
```

Comment:
I quite like what we do for SCP with checking status of the resources with the one liners. It removes the need for explicitly showing what it should look like.
```suggestion
If everything went well, the status of the resource should be `Ready`

```bash
kubectl get kuadrant kuadrant -n kuadrant-system -o=jsonpath='{.status.conditions[?(@.type=="Ready")].message}{"\n"}'```

```